how-to-docs/certificates.md

```diff
@@ -1,13 +1,34 @@
 ### **Application Certificates**
 -----
+Before reading about certificates and how you can create and manage them. [Please familiarise yourself with our DNS naming convention first](../docs/dns.md)
+
+The platform provides two ways of managing [HTTPS certificates](https://en.wikipedia.org/wiki/HTTPS):
+- [Internal based certificates](#internal-certificates) i.e. `hostname.namespace.svc.cluster.local` using CFSSL
+- [External based certificates](#external-certificates) for external services i.e. `service.homeoffice.gov.uk` using kube-cert-manager and letsencrypt
+
+In most systems, it's likely that your service will have a user facing service, that will be served through an external endpoint i.e. it can be routed to externally by users as well as having non-external facing services i.e. internal services.
+
+You would want all communication between the user, through to the service and service dependencies to be encrypted, so that the traffic flow has encryption and that all endpoints trust who they are speaking to.
+
+### Internal Certificates
+
+If your service is not externall facing i.e. there is no external routing to your application,  then you will want an internal certificates for  services that want encryption of traffic between one another, but with an internal trusted CA. This means that serviceA can talk to serviceB securely and trust the authority that has issued the certificates for both of those services. We use CFSSL to sign the CSR's and have tools that you will place with your deployment, that will enable the creation and storage of the certificate for your webserver or application.
```

Comment:
`externall` => `external`

Reply:
`internal certificates for  services` => `internal certificate for services` (extra s and extra space)